WIP - MTA-4637: Pattern matching docs need urgent improvements (#92)
migtools/mta-documentation: docs/topics/yaml-message-actions.adoc

// Module included in the following assemblies:
//
// * docs/rules-development-guide/master.adoc

:_mod-docs-content-type: REFERENCE
[id="yaml-message-actions_{context}"]
= Message action

> A message action is employed to generate an issue with the specified message when a rule matches:
> 
> For example:

Reviewer: ```suggestion
A message action is employed to generate an issue with the specified message when a rule matches, for example:
```

[source,yaml]
----
# When a match is found, the analyzer generates incidents with the same message.
message: "helpful message about the violation"
----

> It is also possible to create a template message to include information about the match that has been interpolated through custom variables on the rule.

Reviewer: ```suggestion
You can also create a template message to include information about the match that has been interpolated through custom variables on the rule.
```
